pandas/tests/groupby/test_transform.py

```diff
@@ -1074,3 +1076,58 @@ def test_transform_lambda_with_datetimetz():
         name="time",
     )
     assert_series_equal(result, expected)
+
+
+def test_transform_cumcount_ngroup():
+    df = DataFrame(dict(a=[0, 0, 0, 1, 1, 1], b=range(6)))
+    g = df.groupby(np.repeat([0, 1], 3))
+
+    # GH 27472
+    result = g.transform("cumcount")
+    expected = g.cumcount()
+    assert_series_equal(result, expected)
+
+    # GH 27468
+    result = g.transform("ngroup")
+    expected = g.ngroup()
+    assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "func",
+    [
+        "backfill",
```

Comment:
Is there not already a fixture for these we can leverage?

Reply:
ok, will change! thanks for review!